e2e/wdio/platform/pages/combobox.po.ts

```diff
@@ -34,19 +34,19 @@ export class ComboBoxPo extends BaseComponentPo {
     };
 
     comboBoxButtons = (name: string) => {
-        return `//label[@for='${name}']/following-sibling::fdp-input-message-group//button`;
+        return `//label[@for='${name}']/../../fdp-input-message-group//button`;
```

Comment:
interesting, is this working?

Reply:
This one was me))) Yes, it works)

Reply:
basically, the structure of the page changed a little and fdp-input-group-button was no longer a sibling of label. So I needed to get label's grandparent to grab  fdp-input-group-button.